aws/sdk/build.gradle.kts

```diff
@@ -196,27 +202,29 @@ fun generateSmithyBuild(tests: List<AwsService>): String {
                 ""
             )
         }
+        val eventStreamAllowListMembers = eventStreamAllowList.joinToString(", ") { "\"$it\"" }
         """
             "${it.module}": {
                 "imports": [${files.joinToString()}],
 
                 "plugins": {
                     "rust-codegen": {
-                      "runtimeConfig": {
-                        "relativePath": "../"
-                      },
-                      "codegen": {
-                        "includeFluentClient": false,
-                        "renameErrors": false
-                      },
-                      "service": "${it.service}",
-                      "module": "aws-sdk-${it.module}",
-                      "moduleVersion": "${getProperty("aws.sdk.version")}",
-                      "moduleAuthors": ["AWS Rust SDK Team <aws-sdk-rust@amazon.com>", "Russell Cohen <rcoh@amazon.com>"],
-                      "license": "Apache-2.0"
-                      ${it.extraConfig ?: ""}
-                 }
-               }
+                        "runtimeConfig": {
+                            "relativePath": "../"
+                        },
+                        "codegen": {
+                            "includeFluentClient": false,
+                            "renameErrors": false,
+                            "eventStreamAllowList": [$eventStreamAllowListMembers]
```

Comment:
is this a list of services? Or a list of operations? Can this just be a boolean flag?

Reply:
It's a list of services currently, and this PR is populating it with `aws-sdk-s3` and `aws-transcribestreaming`.